apps/fhir/authentication.py

```diff
@@ -0,0 +1,25 @@
+import base64
+
+
+def extract_username(auth):
```

Comment:
Can you make this name a little more specific? Maybe extract_username_from_sls_header?